### DISCUSSION TITLE
Proposal to update the personalization section

### DISCUSSION THREAD

**Discussion by wesleymiles** [RedHat-UX/red-hat-design-system#1203] [General]
hi, i propose to update the personalization section of ux dot to better reflect its current state and to build a foundation to scale it with more patterns. it hasn't been updated to keep up with the changes on the pzn team.

this "project" has floundered in Corey, Dustin, and my task lists' for some time, but now we've focused our vision to ship a small-scoped update, including:
- updating/creating an overview page with relevant content 
- 3 patterns to begin with (announcement, sticky card, and for you)
- give it it's own section in the nav 
- think of this version as representative of our current state, an MVP, resisting the urge to expand

Post-MVP could be more patterns, links to live versions of past implementations, etc.

here's a rough outline of the content: https://docs.google.com/presentation/d/1LX4FTo2ojxbXOojY_XCuqm-brdwt8KPqJcnezj2W0Sg/edit#slide=id.g859e711758_4_48

and an image of how we may organize this section in the nav, and lay out the content on an individual pattern page. 
![Announcement – 2](https://github.com/RedHat-UX/red-hat-design-system/assets/613815/c590f98c-3366-4823-a7ad-d50df25f943f)

the three of us can continue working on the meat of the changes.

**Comment by bennypowers** [comment]
[*challenge accepted*](https://github.com/RedHat-UX/red-hat-design-system/issues/1204) :tophat:
